.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,26 @@
 ## Purpose
-Explain the goal of this PR, if it addresses an existing issue be sure to link to it. Describe the big picture of your changes here, perhaps using a bullet list if multiple changes are done to accomplish a single goal.  If it accomplishes multiple goals, it would be best to create separate PR's for each. 
+Explain the goal of this PR, if it addresses an existing issue be sure to link to it. Describe the big picture of your changes here, perhaps using a bullet list if multiple changes are done to accomplish a single goal. If it accomplishes multiple goals, it may be best to create separate PR's for each. 
```

Comment:
If the default MDO Lab templates [here](https://github.com/mdolab/.github/tree/master/.github) are okay, then we can just delete this file and GitHub will automatically use the default organizational template. Same with the ISSUE template.

Reply:
Yes, will delete the template from OAS repo then.